src/helix-ui/styles/vars.less

```diff
@@ -92,10 +92,11 @@
 @app-nav-width: 15rem;
 
 // ==== Z-INDEX =====
+@drawer-z-index: 50;
 @menu-z-index: 100;
 @search-assistance-z-index: 100;
-@tooltip-z-index: 1000;
-@popover-z-index: 1100;
+@popover-z-index: 1000;
+@tooltip-z-index: 1100;
```

Comment:
As above:
```suggestion
@tooltip-z-index: 1000;
```

Reply:
This was intentional. See comment above.